src/server.cpp

```diff
@@ -291,26 +288,18 @@ Server::Server(
 	}
 	infostream << std::endl;
 	// Load and run "mod" scripts
-	for (std::vector<ModSpec>::iterator i = m_mods.begin();
-			i != m_mods.end(); ++i) {
-		const ModSpec &mod = *i;
+	for (std::vector<ModSpec>::iterator it = m_mods.begin();
+			it != m_mods.end(); ++it) {
+		const ModSpec &mod = *it;
```

Comment:
I prefer `it` because `i` is traditionaly an abbreviation for `integer`.  This doesn't really matter much though, I wouldn't mind removing it.